src/connectors/connector_cloudscheduler.workflows.yaml

```diff
@@ -24,7 +24,7 @@
       - project_id: ${sys.get_env("GOOGLE_CLOUD_PROJECT_ID")}
       - location_id: "us-central1"
       - job_id: "dummy-job-id"
-      - topic_id: "dummy-topic-id"  # Pubsub topic used by Cloud Scheduler as a notification target.
+      - topic: "dummy-topic-id"  # Pubsub topic used by Cloud Scheduler as a notification target.
```

Comment:
Pre-existing, but this feels like an inclusive language bug.

Reply:
https://github.com/GoogleCloudPlatform/workflows-samples/issues/182